ghost/core/core/server/data/migrations/versions/5.16/2022-09-19-17-44-add-referrer-attribution-columns-to-events-table.js

```diff
@@ -0,0 +1,39 @@
+const {createAddColumnMigration, combineNonTransactionalMigrations} = require('../../utils');
+
+module.exports = combineNonTransactionalMigrations(
+    createAddColumnMigration('members_created_events', 'attribution_ref_source', {
+        type: 'string',
+        maxlength: 191,
+        nullable: true
+    }),
+
+    createAddColumnMigration('members_created_events', 'attribution_ref_medium', {
+        type: 'string',
+        maxlength: 191,
+        nullable: true
+    }),
+
+    createAddColumnMigration('members_created_events', 'attribution_ref_url', {
+        type: 'string',
+        maxlength: 2000,
+        nullable: true
+    }),
+
+    createAddColumnMigration('members_subscription_created_events', 'attribution_ref_source', {
+        type: 'string',
+        maxlength: 191,
+        nullable: true
+    }),
+
+    createAddColumnMigration('members_subscription_created_events', 'attribution_ref_medium', {
+        type: 'string',
+        maxlength: 191,
+        nullable: true
+    }),
+
+    createAddColumnMigration('members_subscription_created_events', 'attribution_ref_url', {
+        type: 'string',
+        maxlength: 2000,
+        nullable: true
+    })
+);
```

Comment:
@daniellockyer Does all the columns in single migration work better here or should i break this into two separate migrations for two tables ?

Reply:
Although we have no strict rules here, I think it might be better to keep it to 1 migration, 1 table

Therefore, maybe let's split them apart?

Reply:
Yea sounds good, updating

Reply:
Updated now ✅